build_runner/bin/build_runner.dart

```diff
@@ -95,7 +95,10 @@ Future<void> main(List<String> args) async {
   if (localCommandNames.contains(commandName)) {
     exitCode = await commandRunner.runCommand(parsedArgs) ?? 1;
   } else {
-    while ((exitCode = await generateAndRun(args)) == ExitCode.tempFail.code) {}
+    var experiments =
+        (parsedArgs.command!)['enable-experiment'] as List<String>?;
```

Comment:
Are these parens needed?
```suggestion
        parsedArgs.command!['enable-experiment'] as List<String>?;
```